pkg/platform/kube/functionres/lazy.go

```diff
@@ -2149,6 +2150,10 @@ func (lc *lazyClient) populateDeploymentContainer(ctx context.Context,
 	} else {
 		container.ImagePullPolicy = function.Spec.ImagePullPolicy
 	}
+
+	if function.Spec.EnvFrom != nil {
+		container.EnvFrom = function.Spec.EnvFrom
+	}
```

Comment:
Also move up to below `Env` setting.
I think it is better to keep everything envvar-related together to better maintain it.